tests/pc/test_po_update.py

```diff
@@ -99,7 +99,10 @@ def test_po_update(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_fro
             portchannel_members = port_channels_data[portchannel]
             break
 
-    pytest_assert(portchannel and portchannel_members, 'Can not get PortChannel interface for test')
+    if not portchannel:
```

Comment:
if portchannel is not present wont this check in line 90 skip the test?
```
 port_channels_data = asichost.get_portchannels_and_members_in_ns(tbinfo)
    if not port_channels_data:
        pytest.skip(
            "Skip test as there are no port channels on asic {} on dut {}".format(enum_frontend_asic_index, duthost))
```

Reply:
Yes, you're right. This patch predated https://github.com/sonic-net/sonic-mgmt/pull/7574 which merged last year. So, this PR is not needed.

Reply:
Thanks @kenneth-arista, can you please close this PR